backend/package.json

```diff
@@ -18,11 +19,18 @@
   },
   "homepage": "https://github.com/uwblueprint/water-festival#readme",
   "dependencies": {
+    "bluebird": "^3.5.1",
     "body-parser": "^1.18.2",
     "cors": "^2.8.4",
     "dotenv": "^5.0.0",
     "express": "^4.16.1",
     "fuse.js": "^3.1.0",
+    "mongodb": "^3.0.2",
```

Comment:
Do we need this dependency?

Reply:
Yeah, we use it here https://github.com/uwblueprint/water-festival/blob/12c252f3802233846d146e58398818cdb52b7745/backend/controllers/Events.js#L4